pkg/core/state/account.go

```diff
@@ -27,6 +27,33 @@ type UnclaimedBalance struct {
 // UnspentBalances is a slice of UnspentBalance (mostly needed to sort them).
 type UnspentBalances []UnspentBalance
 
+// NEP5Tracker contains info about a single account in a NEP5 contract.
+type NEP5Tracker struct {
+	// Balance is the current balance of the account.
+	Balance int64
+	// LastUpdatedBlock is a number of block when last `transfer` to or from the
+	// account occured.
+	LastUpdatedBlock uint32
+	// Received is a list of received transfers.
+	Received []NEP5Transfer
```

Comment:
I think we're better store these logs (receive/sent) separately from the Account structure.